### PR TITLE
Add additional grid option for expanded AOI

### PR DESCRIPTION
**Pull request by JM-GA**
Sometime later this year an expanded AOI may be rolled out. To anticipate this, a new 30m extended grid option has been added.

`mypy` is disabled temporarily for a further investigation.

### CODE REVIEW

File: odc/dscache/tools/tiling.py [hunk 1]
```diff
@@ -40,6 +40,21 @@
         )
         for n in (10, 20, 30, 60)
     },
+    "au_extended": GridSpec(
+        crs=epsg3577,
+        tile_size=(96_000.0, 96_000.0),
+        resolution=(-96_000, 96_000),
+        origin=(-4416000.0, -6912000.0),
```
Comment (robbibt) [inline odc/dscache/tools/tiling.py]: GridSpec origins are Y, X, so I think this needs to be reversed:

```suggestion
        origin=(-6912000.0, -4416000.0),
```

File: odc/dscache/tools/tiling.py [hunk 2]
```diff
@@ -40,6 +40,21 @@
...
+    ),
+    **{
+        f"au_extended_{n}": GridSpec(
+            crs=epsg3577,
+            tile_size=(96_000.0, 96_000.0),
+            resolution=(-n, n),
+            origin=(-4416000.0, -6912000.0),
```
Comment (robbibt) [inline odc/dscache/tools/tiling.py]: Same here:

```suggestion
        origin=(-6912000.0, -4416000.0),
```

**Review by robbibt — CHANGES_REQUESTED** [review]

**Review by robbibt — APPROVED** [review]
Looks great - have tested the updated geobox and the 0, 0 tile is where I would expect:

image

```
from datacube.model import GridSpec
import geopandas as gpd

gs = GridSpec(
        crs='epsg:3577',
        tile_size=(96_000.0, 96_000.0),
        resolution=(-96_000, 96_000),
        origin=(-6912000.0, -4416000.0))

gpd.GeoSeries(gs.tile_geobox([0, 0]).geographic_extent.geom).to_file("test.geojson")
```

**Comment by JM-GA** [comment]
Thanks @robbibt !

**Review by emmaai — APPROVED** [review]

**Pull request merged**